sql/301_Grafana_Mapping_Create_table.sql

```diff
@@ -0,0 +1,11 @@
+--- Create Grafana Dashboard Mapping Table ---
+drop table if exists public.grafana_dashboard_mapping;
+create table public.grafana_dashboard_mapping
+(
+  org_id int not null,
```

Comment:
Was ist wenn wir mehrere Grafana Systeme haben?
Brauchen wir zu den beiden Tabellen Inhalten oben noch ein "grafana_host" Eintrag in welchem z.B. die IP Adresse des anfragenden Grafana Server steht? 
Oder bekommt jeder Grafana Server eine eigene mapping Tabelle?